drakrun/drakrun/storage.py

```diff
@@ -307,6 +336,11 @@ def export_vm0(self, path: str):
     def import_vm0(self, path: str):
         shutil.copy(path, os.path.join(VOLUME_DIR, 'vm-0.img'))
 
+    def delete_vm_volume(self, vm_id: str) -> bool:
```

Comment:
Don't fail with bool, raise an exception

Reply:
Yeah sure, will update this also